docs/metrics.md

```diff
@@ -0,0 +1,12 @@
+# Build Controller Metrics
+
+The Build component exposes several metrics to help you monitor the health and behavior of your build resources.
+
+Following build metrics are exposed at service `build-operator-metrics` on port `8383`.
+
+| Name | Type | Description | Label | Status |
+| ---- | ---- | ----------- | ----- | ------ |
+| `build_builds_registered_total` | Counter | Number of total created Builds. | buildstrategy=<build_buildstrategy_name> | experimental |
+| `build_buildruns_completed_total` | Counter | Number of total new triggered BuildRuns. | buildstrategy=<build_buildstrategy_name> | experimental |
```

Comment:
Can you correct the descriptions to `Number of registered Builds.` and `Number of completed BuildRuns.`

Reply:
fix that.